test/Conversion/RTLToLLHD/structure.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: circt-opt -convert-rtl-to-llhd -split-input-file -verify-diagnostics %s | FileCheck %s
+// RUN: circt-opt -convert-rtl-to-llhd -simple-canonicalizer -split-input-file -verify-diagnostics %s | FileCheck %s
```

Comment:
Please keep testcases limited to a specific subsystem.  Making this run the lowering and canonicalizer makes the test more fragile.

Reply:
Sorry, just wanted to show that the old tests pass. I'll pull the canonicalization tests out if we want to go this direction.

Reply:
thx!